recipes/glib/all/conanfile.py

```diff
@@ -198,7 +198,10 @@ def package_info(self):
                 self.cpp_info.components["gio-2.0"].requires.append("libmount::libmount")
             if self.options.with_selinux:
                 self.cpp_info.components["gio-2.0"].requires.append("libselinux::libselinux")
-        if self.settings.os != "Windows":
+        if self.settings.os == "Windows":
+            self.cpp_info.components["gio-windows-2.0"].requres = ["gobject-2.0", "gmodule-2.0", "gio-2.0"]
```

Comment:
```suggestion
            self.cpp_info.components["gio-windows-2.0"].requires = ["gobject-2.0", "gmodule-2.0", "gio-2.0"]
```

Reply:
thanks! even with so few lines, I'm able to make stupid typos...